src/components/ControlPanel/controlPanel.js

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
-
+import Button from 'components/Button';
 
 export default class ControlPanel extends React.Component {
 	render() {
 		return (
+			
 			<div className="control-panel-wrapper">
-				<h1>hello world</h1>
+				<div className="form-elem">
+					<h1>Create a milestone</h1>
+					<textarea
```

Comment:
This should not be a textarea. It should be a text input (single line).
Also the font size is too small. increase the size.